File: escritorio/objective-c/MTWDM - MacOS/AppBindings/AppBindings/ViewController.m
//
//  ViewController.m
//  AppBindings
//
//  Created by Felipe Hernandez on 20/02/21.
//

#import "ViewController.h"

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    // Do any additional setup after loading the view.
}


- (void)setRepresentedObject:(id)representedObject {
    [super setRepresentedObject:representedObject];

    // Update the view, if already loaded.
}

-(IBAction)Seleccionar:(id)sender{
    
    switch ([sender selectedSegment]) {
        case 0:
            [_txtSeleccionado setStringValue:@"Hombre"];
            break;
        case 1:
            [_txtSeleccionado setStringValue:@"Mujer"];
            break;
        case 2:
            [_txtSeleccionado setStringValue:@"Niño"];
            break;
        default:
            break;
    }
}

@end

File: proyectos/conexionesApp/conexionesApp/ViewController.swift
//
//  ViewController.swift
//  conexionesApp
//
//  Created by Marco Cesar Palomino Ravelo on 13/03/21.
//

import UIKit

class ViewController: UIViewController {

    var objEmpleado:Empleado?
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        nombreDepartamento.text = "Depratamento Sistemas"
    }

    @IBOutlet weak var nombreDepartamento: UILabel!
   
    @IBAction func altaEmpleado(_ sender: UIButton) {
        objEmpleado = Empleado()
        objEmpleado?.nombre = "Marco"
        objEmpleado?.apellido = "Palomino"
        objEmpleado?.numeroEmpleado = 991
        objEmpleado?.numeroIMSS = 1234
        
        print("Entro al clicl alta empleado");
    }
    
    @IBAction func consultarEmpleado(_ sender: UIButton) {
        print("entro al consultar empleado")
        print("\(objEmpleado!.nombre)")
        print("\(objEmpleado!.mostrarNombreCompleto())")
        print("\(objEmpleado!.mostrarNumero())")
       
    }
    
    @IBAction func verIms(_ sender: UIButton) {
        print("entro al ver Imss")
        print("\(objEmpleado!.numeroIMSS)")
    }
    
    @IBAction func bajaEmpleado(_ sender: UIButton) {
        print("entro al baja empleado")
    }
    
    
    
}

